mod-tool/src/main/scala/at/forsyte/apalache/tla/tooling/opt/ConfigCmd.scala

```diff
@@ -2,19 +2,72 @@ package at.forsyte.apalache.tla.tooling.opt
 
 import org.backuity.clist.{Command, _}
 import java.io.File
+import com.typesafe.scalalogging.LazyLogging
+// FIXME Why "_root_"?
+import _root_.util.ExecutionStatisticsCollector
+import _root_.util.ExecutionStatisticsCollector.Selection
 
 /**
  * This command initiates the 'config' command line.
  *
  * @author
  *   Igor Konnov
  */
-class ConfigCmd extends Command(name = "config", description = "Configure Apalache options") with General {
+class ConfigCmd
+    extends Command(name = "config", description = "Configure Apalache options") with General with LazyLogging {
 
   var submitStats: Option[Boolean] = opt[Option[Boolean]](name = "enable-stats",
       description = "Let Apalache submit usage statistics to tlapl.us\n"
         + "(shared with TLC and TLA+ Toolbox)\nSee: https://apalache.informal.systems/docs/apalache/statistics.html")
 
   // Used for application (esp. output) configuration
   def file = new File("config")
+
+  def run() = {
+    logger.info("Configuring Apalache")
+    submitStats.foreach { isEnabled =>
```

Comment:
Then we need a separate call to get the value to check if it's enabled. I prefer the `forEach` since the logic will work the same regardless of what container we have here, and in any case this wasn't introduced in this PR, so I'd rather leave it as is.